src/ApiError.php

```diff
@@ -33,20 +33,8 @@ public static function fromRaw($raw)
         }
 
         $error = new static;
-        if (isset($raw->title)) {
-            $error->title = $raw->title;
-        }
-
-        if (isset($raw->detail)) {
-            $error->detail = $raw->detail;
-        }
-
-        if (isset($raw->status)) {
-            $error->status = $raw->status;
-        }
-
-        if (isset($raw->source)) {
-            $error->source = $raw->source;
+        foreach (get_object_vars($raw) as $key => $value) {
```

Comment:
This change is needed so that for that the extra validation details in cluster validation get loaded into the ApiError object